classes/PaymentModule.php

```diff
@@ -198,9 +198,9 @@ public function addCheckboxCarrierRestrictionsForModule(array $shops = [])
      * @param string $payment_method Payment method (eg. 'Credit card')
      * @param string|null $message Message to attach to order
      * @param array $extra_vars
-     * @param null $currency_special
+     * @param int|null $currency_special
      * @param bool $dont_touch_amount
-     * @param bool $secure_key
+     * @param string|bool $secure_key
```

Comment:
I'm afraid, it should be a bool everytime :)

Reply:
But in `classes/Cart.php` l.80 it's a string

Reply:
In fact you're right, both are accepted -_-